local-test-infra/local-partner-pod.yaml

```diff
@@ -5,7 +5,9 @@ metadata:
   labels:
     app: partner
     test-network-function.com/generic: orchestrator
+    test-network-function.com/container: target
```

Comment:
Not sure I follow what this annotation means?

Reply:
This is equivalent to having the test `partner` listed in place of the `ubuntu` pod in https://github.com/test-network-function/test-network-function/blob/main/test-network-function/tnf_config.yml
It's actually how I have my local config set up, but would make more sense to be the `test` pod in this repo. I'll swap it over to the `test` pod to avoid confusing the roles.